common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -65,7 +65,7 @@
 public class AzureActiveDirectoryWebViewClient extends OAuth2WebViewClient {
     private static final String TAG = AzureActiveDirectoryWebViewClient.class.getSimpleName();
     private static final String INSTALL_URL_KEY = "app_link";
-    private static final String INSTALL_UPN_KEY = "username";
+    public static final String INSTALL_UPN_KEY = "username";
```

Comment:
Should we consider graduating these constants out of this file? Not necessarily that it doesn't belong here (it's set here, after all) but, I notice that now the `MicrosoftStsAuthorizationResultFactory` is reaching-into this class to grab this constant which is maybe a little weird organizationally speaking.

Reply:
Good call, I was being lazy here and just changed the constant to public. Moved to `AuthenticationConstants` as appropriate